src/components/editor/PropertyTemplateOutline.jsx

```diff
@@ -78,15 +100,19 @@ export class PropertyTemplateOutline extends Component {
 
   addPropertyTypeRows = (property) => {
     let propertyJsx, existingJsx, newOutput = this.state.propertyTypeRow
-
     if (isResourceWithValueTemplateRef(property)) {
+      const addButtonDisabled = property.repeatable == "false" ? true : false
```

Comment:
Why is `property.repeatable` a string instead of a boolean?

Reply:
also, ternary operator not necessary in this context per above.

Reply:
(see my comment above)

Reply:
(see all four cases indicated in recent comment above)

Reply:
New `isRepeatable` function in Utilities now handles all four cases in the latest commit.